### PR TITLE
Updated Azure docs to be new portal only

### PR DESCRIPTION
**Pull request by DanielLanger**
Can now integrate with Azure through the new portal only! Docs reflect this change.

### CODE REVIEW

File: content/integrations/azure.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Connect to Microsft Azure in order to:
 
 * Get metrics from Azure VMs with or without installing the Agent
 * Tag your Azure VMs with Azure-specific information (e.g. location)
-* Get metrics for other services: Application Gateway, App Service (Web & Mobile), Batch Service, Event Hub, IOT Hub, Logic App, Redis Cache, SQL Database, SQL Elastic Pool, and Virtual Machine Scale Set
+* Get metrics for other services: Application Gateway, App Service (Web & Mobile), Server Farm, Batch Service, Event Hub, IOT Hub, Logic App, Redis Cache, SQL Database, SQL Elastic Pool, and Virtual Machine Scale Set
```
Comment (technovangelist) [inline content/integrations/azure.md]: everything else in the list seems to be alpha sorted, except for server farm. is that intentional?

File: content/integrations/azure.md [hunk 2]
```diff
@@ -103,62 +101,73 @@ azure role assignment create --objectId {object-Id} --roleName Reader --subscrip
...
-1. Navigate to [portal.azure.com][2]
-2. Navigate to “Subscriptions” on the left hand menu
+1. Navigate to “Subscriptions” on the left hand menu
 
     ![settings](/static/images/azure/subscriptions_icon.png){:style="width:50%;"}
 
-3. Click on the subscription you would like to monitor
-4. Click on "Access control (IAM)" in the lefthand menu 
+2. Click on the subscription you would like to monitor
+3. Click on "Access control (IAM)" in the lefthand menu
```
Comment (DanielLanger) [inline content/integrations/azure.md]: 👍  think it was from a larger chunk of code I deleted and reordered. Looks good on Rake

File: content/integrations/azure.md (hunk 1 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: 👍  fixed

**Comment by technovangelist** [comment]
looks good to go, thanks

**Pull request merged**